qiskit/synthesis/multi_controlled/mcx_with_ancillas_synth.py

```diff
@@ -0,0 +1,203 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Module containing multi-controlled circuits synthesis with ancillary qubits."""
+
+from math import ceil
+
+
+def synth_mcx_n_dirty_i15(
+    num_ctrl_qubits: int,
+    relative_phase: bool = False,
+    action_only: bool = False,
+):
```

Comment:
Could you please describe the arguments to this function in the docstring (e.g., what are `relative_phase` and `action_only`)?